crates/ere-jolt/src/lib.rs

```diff
@@ -24,8 +26,14 @@ impl Compiler for JOLT_TARGET {
 
     type Program = Program;
 
-    fn compile(path_to_program: &std::path::Path) -> Result<Self::Program, Self::Error> {
-        let manifest_path = path_to_program.to_path_buf().join("Cargo.toml");
+    fn compile(
+        workspace_directory: &Path,
+        guest_relative: &Path,
+    ) -> Result<Self::Program, Self::Error> {
+        let manifest_path = workspace_directory
+            .join(guest_relative)
```

Comment:
Since today only SP1 is the one dockerized, for all zkVMs that aren't SP1 you'll see that they simply join the workspace directory with the guest relative, since this is equivalent to the previous `compile` API.